cdap-app-templates/cdap-etl/cdap-etl-batch/src/test/java/co/cask/cdap/template/etl/batch/BatchETLDBAdapterTest.java

```diff
@@ -196,7 +196,9 @@ public void testDBSource() throws Exception {
     ETLStage sink = new ETLStage("Table", ImmutableMap.of(
       "name", "outputTable",
       Properties.Table.PROPERTY_SCHEMA, schema.toString(),
-      Properties.Table.PROPERTY_SCHEMA_ROW_FIELD, "ID"));
+      Properties.Table.PROPERTY_SCHEMA_ROW_FIELD, "id",
+      // Test case-insensitive row key matching
+      Properties.Table.CASE_INSENSITIVE_ROW_FIELD, "true"));
```

Comment:
this approach seems weird, so I have to name my columns accordingly, but not the row field? kind of defeats the overall goal right?